target/scripts/startup/setup-stack.sh

```diff
@@ -1183,6 +1183,44 @@ EOF
   supervisorctl update
 }
 
+function _setup_getmail
+{
+  _log 'trace' 'Preparing Getmail configuration'
+
+  local CONFIGURATION GETMAILRC ID CONFIGS
+
+  CONFIGURATION='/tmp/docker-mailserver/getmail-*.cf'
```

Comment:
We do not need `CONFIGURATION` to be in a variable really, just use idiomatic globbing for the loop:

```suggestion
  local GETMAILRC ID CONFIGS

```